### ISSUE TITLE
AI for Good

### ISSUE THREAD

**Issue by kremalicious** [oceanprotocol/commons#123]
- [x] new category: "AI for Good"
- [x] add new curated assets into commons (@sheriff4)
- [x] tag those new assets manually in database with `ai-for-good`
- [x] provide copy for AI for Good channel: teaser, description (@sheriff4)
- [x] add AI Commons logo in footer (#137)
- [x] front-page gets link to "AI for Good" category/visual list of assets from that category (#125)
- [x] show total number of assets somewhere, "online since..." (#128)
- [ ] descending search results order, sorted by date published (blocked by https://github.com/oceanprotocol/aquarius/issues/179 & https://github.com/oceanprotocol/aquarius/issues/180)
- [x] new banner image for asset details view, default to category image
- [x] (nice to have) implement adding custom images for an asset

until May 28.
